src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -50,7 +50,10 @@
             <a href="#" class="review-history-show" data-div="#{{ version.id }}-review-history">{{ _('Review History') }}</a>
             <a href="#" class="review-history-hide hidden">{{ _('Close Review History') }}</a>
             {% if is_last_version %}
-                <b class="review-history-pending-count">{{ pending_activity_log_count_for_developer(version) }}</b>
+            {% set pending_count = pending_activity_log_count_for_developer(version) %}
+                {% if pending_count>0 %}
+                <b class="review-history-pending-count">{{ pending_count }}</b>
```

Comment:
This should be indended one more level to be inside the `if pending_count` block